.config/ansible-lint.yml

```diff
@@ -2,5 +2,9 @@
 profile: production
 
 exclude_paths:
+  - .ansible/
   - tests/integration
+  - tests/unit
   - tests/sanity
+  - meta/
```

Comment:
The `meta/` directory needs to not be ignored. There are rules for `meta/runtime.yml` that we need to make sure are run.

Reply:
> The `meta/` directory needs to not be ignored. There are rules for `meta/runtime.yml` that we need to make sure are run.

Removed from exclude